selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -237,18 +181,58 @@ private Locators() {}
     }
   }
 
-  @FindBy(id = Locators.EDITOR_TABS_PANEL)
-  private WebElement editorTabsPanel;
+  /** Editor`s markers types */
+  public enum MarkersType {
```

Comment:
It seems type of ordinary type, not a bunch of types. So, let's name it `Marker`.

Reply:
No, this is exactly different marker's types. But I agree that name is not so very accurate. I'll change it to "MarkerTypes".